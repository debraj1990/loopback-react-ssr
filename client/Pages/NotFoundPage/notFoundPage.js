import React from 'react';

export const NotFoundPage = ({ staticContext = {} }) => {
  staticContext.notFound = true;
  return <h1>Ooops, route not found.</h1>;
};

export default {
  component: NotFoundPage
};
