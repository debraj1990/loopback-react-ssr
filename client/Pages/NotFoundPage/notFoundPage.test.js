import React from 'react';
import {configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16';

import { NotFoundPage } from './notFoundPage';


configure({adapter: new Adapter });


describe('<NotFoundPage />', () => {
    

    it('should render the NotFoundPage', () => {
        const component = shallow(<NotFoundPage />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })

    it('should contain h1 text of opps..', () => {
        const component = shallow(<NotFoundPage />)
        expect(component.contains(<h1>Ooops, route not found.</h1>)).toBeTruthy()
    })
})