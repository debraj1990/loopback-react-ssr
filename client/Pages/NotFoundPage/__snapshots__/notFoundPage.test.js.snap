// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render the NotFoundPage 1`] = `
<h1>
  Ooops, route not found.
</h1>
`;
